src/components/OrderTable/OrderTable.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import {
+  Button,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import classes from './OrderTable.module.scss';
+
+interface OrderTableProps {
+  rows: Row[];
+}
+
+interface Row {
+  name: string;
+  img: string;
+  qty: number;
+  price: number;
+}
+
+const TAX_RATE = 0.2;
+
+function ccyFormat(num: number) {
+  return `${num.toFixed(2)}`;
+}
+
+function priceRow(qty: number, price: number) {
+  return qty * price;
+}
+
+function total(items: readonly Row[]) {
+  return items
+    .map(({ qty, price }) => priceRow(qty, price))
+    .reduce((sum, i) => sum + i, 0);
+}
+
+const OrderTable: React.FC<OrderTableProps> = ({ rows }) => {
+  const invoiceTotal = total(rows);
+  const invoiceSubtotal = (invoiceTotal / (TAX_RATE * 100 + 100)) * 100;
+  const invoiceTaxes = invoiceTotal - invoiceSubtotal;
```

Comment:
I think that should be more clear
```suggestion
  const invoiceTaxes = invoiceTotal * TAX_RATE;
  const invoiceSubtotal = invoiceTotal - invoiceTaxes;
```

Reply:
fixed